### DISCUSSION TITLE
MenuSeparator not being displayed

### DISCUSSION THREAD

**Discussion by guiribmedeiros** [laravel/nova-issues#5015] [Ideas & Feature Requests] [answered]
- Laravel Version: 9.37.0
- Nova Version: 4.17.1
- PHP Version: 8.0.24
- Operating System and Version: macOS Monterey 12.6.1
- Browser type and version: Google Chrome 106.0.5249.119

### Description:
- The `MenuSeparator` is not working
- It is being rendered, but invisible

### Detailed steps to reproduce the issue on a fresh Nova installation:
- Just create a custom `mainMenu`
- Add the `MenuSeparator::make()` component
- It won't be displayed
- Remove the `border-none` class from the `hr` tag and it will work again

**Comment by davidhemphill** [comment, marked as answer]
Moving to a discussion since `MenuSeparator` isn't documented and intended for use.
